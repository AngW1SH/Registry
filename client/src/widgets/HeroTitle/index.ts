import HeroTitle from "./ui/HeroTitle";

export { HeroTitle };
