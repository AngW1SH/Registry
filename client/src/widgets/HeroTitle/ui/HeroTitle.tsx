import { FC } from "react";

interface HeroTitleProps {}

const HeroTitle: FC<HeroTitleProps> = () => {
  return (
    <h1 className="text-center text-4xl text-white sm:text-7xl md:text-left">
      Реестр проектов
    </h1>
  );
};

export default HeroTitle;
