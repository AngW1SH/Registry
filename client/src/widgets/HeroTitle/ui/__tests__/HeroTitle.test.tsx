import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import HeroTitle from "../HeroTitle";

describe("Hero Title Widget UI", () => {
  it("displays the title", () => {
    const { getByText } = render(<HeroTitle />);

    expect(getByText(/Реестр проектов/)).toBeInTheDocument();
  });

  test("Snapshot test", () => {
    const { asFragment } = render(<HeroTitle />);

    expect(asFragment()).toMatchSnapshot();
  });
});
