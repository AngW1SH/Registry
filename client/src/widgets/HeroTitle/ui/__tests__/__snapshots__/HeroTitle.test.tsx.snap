// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Title Widget UI Snapshot test 1`] = `
<DocumentFragment>
  <h1
    class="text-center text-4xl text-white sm:text-7xl md:text-left"
  >
    Реестр проектов
  </h1>
</DocumentFragment>
`;
